test/common/components/RequiredLoginView.unit.spec.jsx

```diff
@@ -98,10 +104,28 @@ describe('<RequiredLoginView>', () => {
     );
     tree.getMountedInstance().setState({
       loading: false,
-      isServiceAvailableForUse: false
     });
 
-    // each direct child should get their props.isDataAvailable set
+    // Child components should not be passed an isDataAvailable prop
+    tree.props.children.forEach((child) => {
+      expect(child.props.isDataAvailable).to.be.undefined;
```

Comment:
Not your job since this behavior existed before you refactored, but I've logged an issue to address the fact that the possible values for `isDataAvailable` is `undefined` & `false`.  I think ideally, it should be `true` or `false`.

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2752